cabal-install/src/Distribution/Client/Init/Interactive/Command.hs

```diff
@@ -139,9 +139,10 @@ createProject v pkgIx srcDb initFlags = do
         (Just exeTarget) testTarget
     
     TestSuite -> do
-      testTarget <- genTestTarget initFlags pkgIx
+      let initFlags' = initFlags { initializeTestSuite = Flag True }
```

Comment:
Just an addendum: this is necessary in `--interactive` because if both package type and test flags are **not** passed, the user will be prompted for a package type, and it now includes `TestSuite` in the list. In short, that means you could have a `TestSuite` target, with `initializeTestSuite` set to `NoFlag`, which would prompt the user if they want a test suite or not, considering that the target is `TestSuite`, that prompt behaviour is undesired.

Reply:
That would make a great comment IMHO.

Reply:
I actually just thought of that, I shall make it so.